openlibrary/core/schema.sql

```diff
@@ -52,6 +52,17 @@ INSERT INTO bookshelves (name, description) VALUES ('Want to Read', 'A list of b
 INSERT INTO bookshelves (name, description) VALUES ('Currently Reading', 'A list of books I am currently reading');
 INSERT INTO bookshelves (name, description) VALUES ('Already Read', 'A list of books I have finished reading');
 
+CREATE TABLE bookshelves_events (
+    id serial primary key,
+    username text not null,
+    work_id integer not null,
+    edition_id integer not null,
+    event_type integer not null,
```

Comment:
Is there a reason we didn't use postgres enums here?

See https://www.postgresql.org/docs/current/datatype-enum.html

Reply:
We use MySql, which handles enums differently, for testing.